frontend/views/containers/payments/PaymentsMixin.js

```diff
@@ -69,44 +73,40 @@ const PaymentsMixin: Object = {
       return await sbp('gi.db/archive/load', ourArchiveKey) ?? {}
     },
 
-    async getHistoricalPaymentsInTypes () {
-      const paymentsInTypes = {
-        sent: cloneDeep(this.ourPayments?.sent || []),
-        received: cloneDeep(this.ourPayments?.received || []),
-        todo: cloneDeep(this.ourPayments?.todo || [])
-      }
-      const paymentsByPeriod = await this.getHistoricalPaymentPeriods()
+    async getAllPaymentsInTypes () {
+      const sent = []
+      const received = []
```

Comment:
Why did this remove `cloneDeep(this.ourPayments?.sent || [])` and `cloneDeep(this.ourPayments?.received || [])`?

Reply:
To avoid collecting the same entries twice since the current period is going to be processed anyway in the below loop. Another idea would be to keep the `cloneDeep` calls, but then exclude the current period from the loop.

Reply:
I see..